charts/cinder-csi-plugin/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: latest
 description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 1.2.0
+version: 1.2.2
```

Comment:
just curious on the bump from .0 to .2 , any reason behind it?

Reply:
@jichenjc this PR , https://github.com/kubernetes/cloud-provider-openstack/pull/1211 , bumps to 1.2.1